### PR TITLE
web viewer configuration

### PR DESCRIPTION
**Pull request by will-moore**
This adds documentation for https://github.com/openmicroscopy/openmicroscopy/pull/4480.

Can test the docs as described, using the webtest example with the latest code from https://github.com/openmicroscopy/webtest/pull/8

### CODE REVIEW

**Comment by jburel** [comment]
- `:property:’omeroweb.webclient.views.image_viewer’` does not render
- in other sections of the page ref to url.py view.py are in bold
- ‘channel_overlay_viewer’ usage of double quote elsewhere in the page.
- Code snippets do not render well

## See screenshot below:

customimageviewer

**Comment by hflynn** [comment]
New properties don't render correctly until they have been added in a code PR and the properties page has been updated by the autogen branch. The code blocks issue is due to too much indenting.

File: omero/developers/Web/CreateApp.txt [hunk 1]
```diff
@@ -358,6 +358,40 @@ of links at the top of the webclient main pages.
...
+Custom image viewer
+^^^^^^^^^^^^^^^^^^^
+
+If you have created your own image viewer and would like to have it replace the existing image viewer in the
+webclient, this can be configured using :property:'omeroweb.webclient.views.image_viewer'.
+
+You will need your views.py method to take an Image ID with a parameter named ``iid``. For example, see
+``channel_overlay_viewer`` from the `webtest <https://github.com/openmicroscopy/webtest/>`_ app:
+
+   ::
```
Comment (hflynn) [inline omero/developers/Web/CreateApp.txt]: If you put the `::` at the end of the sentence and just indent the code snippets by 4 spaces, you'll get rid of the purple highlighting in plone that JM has pointed out.

File: omero/developers/Web/CreateApp.txt [hunk 2]
```diff
@@ -358,6 +358,40 @@ of links at the top of the webclient main pages.
        $ bin/omero config append omero.web.ui.top_links '["Homepage", "http://myhome.com", {"title": "Homepage", "target": "new"}]'
 
 
+Custom image viewer
+^^^^^^^^^^^^^^^^^^^
+
+If you have created your own image viewer and would like to have it replace the existing image viewer in the
+webclient, this can be configured using :property:'omeroweb.webclient.views.image_viewer'.
```
Comment (hflynn) [inline omero/developers/Web/CreateApp.txt]: Ah, the property isn't rendering because you need to use ` backticks quotes not '

**Comment by will-moore** [comment]
That commit should fix the formatting issues above.

**Comment by hflynn** [comment]
Formatting issues are indeed fixed thanks.

**Comment by jburel** [comment]
Instruction testing using docker.
It works as expected. Will review the page again tomorrow

**Comment by jburel** [comment]
I have not found other instances of web mobile in the doc.

**Comment by jburel** [comment]
Not related to the initial change but I think the link to webtagging should point to https://github.com/MicronOxford/webtagging not to douglas account (Create App section)

**Comment by jburel** [comment]
"Create and checkout a new GitHub" should be "check out"

**Comment by jburel** [comment]
The changes now render correctly. Small edits then we should be good

**Comment by hflynn** [comment]
Oh, I'll check plone page for the old webtagging link (if you could fix it here please @will-moore ) incase I've missed some other references to it.

File: omero/developers/Web/CreateApp.txt [hunk 3]
```diff
@@ -358,6 +358,40 @@ of links at the top of the webclient main pages.
        $ bin/omero config append omero.web.ui.top_links '["Homepage", "http://myhome.com", {"title": "Homepage", "target": "new"}]'
 
 
+Custom image viewer
+^^^^^^^^^^^^^^^^^^^
+
+If you have created your own image viewer and would like to have it replace the existing image viewer in the
+webclient, this can be configured using :property:`omeroweb.webclient.views.image_viewer`.
```
Comment (hflynn) [inline omero/developers/Web/CreateApp.txt]: Okay, this property link still isn't working - according to the text added by the autogen build from your code PR, the property is `omero.web.viewer.view` and `omeroweb.webclient.views.image_viewer` is the default setting instead.

**Comment by will-moore** [comment]
Thanks @hflynn - my mistake. Fixed property now.

**Comment by hflynn** [comment]
Looks good to me, good to merge for you too @jburel ?

**Comment by jburel** [comment]
yes, merging.

**Pull request merged**